ethtxmanager/pgstorage.go

```diff
@@ -123,6 +123,50 @@ func (s *PostgresStorage) GetByStatus(ctx context.Context, owner *string, status
 	return mTxs, nil
 }
 
+// GetBySenderAndStatus loads all monitored txs of the given sender that match the provided status
+func (s *PostgresStorage) GetBySenderAndStatus(ctx context.Context, sender string, statuses []MonitoredTxStatus, dbTx pgx.Tx) ([]monitoredTx, error) {
```

Comment:
would you mind replacing the `sender` type from `string` to `common.Address`? 
this would force the parameter to always be an Address instead of any string, helping the consumers of this method to understand this is an Address.